launch/tier4_perception_launch/launch/object_recognition/detection/lidar_radar_based_detection.launch.xml

```diff
@@ -1,18 +1,19 @@
 <?xml version="1.0"?>
 <launch>
-  <!-- lidar param -->
+  <!-- LiDAR parameters -->
   <arg name="input/pointcloud"/>
   <arg name="lidar_detection_model" default="centerpoint" description="options: `centerpoint`, `apollo`, `clustering`"/>
   <arg name="use_pointcloud_map" default="true" description="use pointcloud map in detection"/>
   <arg name="use_object_filter" default="true" description="use object filter"/>
   <arg name="use_pointcloud_container" default="false" description="use pointcloud container for detection preprocessor"/>
   <arg name="container_name" default="pointcloud_container"/>
-  <arg name="score_threshold" default="0.20"/>
+  <arg name="lidar_detection_score_threshold" default="0.35"/>
```

Comment:
[Q] Is this change in threshold intended?

Reply:
Yes, I maintenance LiDAR-Radar fusion and set same parameters for Camera-LiDAR-Radar fusion.